basic_bootloader/src/bootloader/gas_helpers.rs

```diff
@@ -14,7 +14,7 @@ pub fn get_resources_for_tx<S: EthereumLikeTypes>(
     intrinsic_gas: usize,
     intrinsic_pubdata: usize,
     intrinsic_native: usize,
-) -> Result<S::Resources, TxError> {
```

Comment:
Can you add a doc line for this function explaining what the return values represent?